projects/pastanaga/src/lib/calendar/date-input.component.html

```diff
@@ -1,7 +1,11 @@
-<div class="pa-date-input" [class.pa-date-error]="dateError">
+<div class="pa-date-input" 
+    [class.pa-date-error]="dateError"
+    [class.input-focused]="focused">
     <pa-input [(ngModel)]="dateInput"
         [placeholder]="datePlaceholder"
         (valueChange)="checkTypedDate($event)"
+        (focus)="onFocus()"
+        (blur)="onBlur()"
```

Comment:
you could use just `onFocus` and use it like
`
(focus)="onFocus(true)"
(blur)="onFocus(false)"
`